README.md

```diff
@@ -103,17 +101,50 @@ require additional configurations applied to
 ### Prerequisites
 
 The following prerequisites are required to use the helm chart:
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported) 
+- See https://github.com/kubernetes/helm for more information.
+- Splunk Enterprise 7.0 or later.
+- A minimum of one Splunk platform index ready to collect the log data. This index will be used for ingesting logs.
+- An HTTP Event Collector (HEC) token and endpoint. See the following topics for more information:
+  * https://docs.splunk.com/Documentation/Splunk/8.2.0/Data/UsetheHTTPEventCollector
+  * https://docs.splunk.com/Documentation/Splunk/8.2.0/Data/ScaleHTTPEventCollector
+- To send data to [Splunk Observability Cloud:](https://docs.splunk.com/Observability/gdi/opentelemetry/install-k8s.html)
+  - [Splunk Access Token](https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html#admin-org-tokens)
+  - [Splunk Realm](https://dev.splunk.com/observability/docs/realms_in_endpoints/)
+- Administrator access to your [Kubernetes cluster](https://kubernetes.io/) and familiarity with your Kubernetes configuration. You must know where your log information is being collected in your Kubernetes deployment.
 
-- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported)
-- [Kubernetes cluster](https://kubernetes.io/)
+### Deploy with Helm 3.0+
 
-- To send data to [Splunk Enterprise/Cloud](https://docs.splunk.com/Documentation/Splunk/8.2.2/Data/UsetheHTTPEventCollector)
-  - HEC Token
-  - HEC Endpoint
+Helm, maintained by the CNCF, allows the Kubernetes administrator to install, upgrade, and manage the applications running in their Kubernetes clusters.  For more information on how to use and configure Helm Charts,  see the Helm [site](https://helm.sh/) and [repository](https://github.com/kubernetes/helm) for tutorials and product documentation. Helm is the only method that the Splunk software supports for installing Splunk Connect for Kubernetes.
+
+To install and configure defaults with Helm:
+
+* Add Splunk chart repo
+```bash
+helm repo add splunk-otel https://splunk.github.io/sck-otel/
+```
+
+* Get values file in your working directory
+
+```bash
+helm show values splunk-otel/sck-otel > values.yaml
+```
+
+* Prepare this Values file. This file has a lot of documentation for configuring Splunk OpenTelemetry Collector for Kubernetes. Look at this [example](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml). Once you have a Values file, you can simply install the chart with by running
+
+```bash
+helm install my-splunk-connect -f my_values.yaml splunk-otel/sck-otel
+```
+
+To learn more about using and modifying charts, see:
+* https://github.com/splunk/sck-otel/tree/main/charts
+* https://docs.helm.sh/using_helm/#using-helm.
+
+
+### Configuration variables for Helm
+
+The default values file can be found here [default values file](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml)
```

Comment:
this part is referencing to old repository. it needs to be updated to this repo.

actually, we already have a simple version of helm install command example for o11y suite. Line # 169 of your branch. 
let's add one in similar format. 

```
helm install my-splunk-otel-collector --set="splunkPlatform.endpoint=127.0.0.1:8088,splunkPlatform.token=xxxxxx,splunkPlatform.metrics_index=k8s-metrics,splunkPlatform.index=main,clusterName=my-cluster" splunk-otel-collector-chart/splunk-otel-collector
```

Reply:
and delete this whole part.

Reply:
moved and deleted - i think this is what you meant?